frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts

```diff
@@ -56,3 +56,6 @@ export const acceptForeverCommunityWarningModal = () =>
     .$('input')
     .click()
     .then(() => acceptCommunityWarningModal());
+export const viewInstalledOperatorsBtn = $('[data-test-id="view-installed-operators-btn"]');
+export const operatorInstallPageLoaded = () =>
+  browser.wait(until.presenceOf(viewInstalledOperatorsBtn), 20000).then(() => browser.sleep(1000));
```

Comment:
Avoid arbitrary sleeps.

Should this wait until the button is clickable instead?

Reply:
This was the only way to get the test to reliably pass, but that was before I switched it to a component within the same install page.  I will fiddle with it some more.

Reply:
Works without the delay now that it is the same page loading a different component.